client/src/pages/guide/english/css/background-size/index.md

```diff
@@ -6,12 +6,35 @@ title: Background Size
 The background-size property specifies the size of the background images. You can set a length or a percentage, with the first value being the width and the second one being the height. You can also use one of the 5 keyword values:
 
 ```css
-.auto {background-size: auto;}
-.cover {background-size: cover;}
-.contain {background-size: contain;}
-.initial {background-size: initial;}
-.inherit {background-size: inherit;}
- /* Percentage and pixel can also be used */
+.auto {background-size: auto;} <!-- The background image is displayed in its original size -->
```

Comment:
```<!--``` is not a valid comment in CSS. You must use ```/*``` and ```*/``` to begin and end comments in CSS.